internal/config/struct.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+type server struct {
+	Host string
```

Comment:
For avoiding the ambiguity from the naming style of config fields, I suggest always adding the json/yaml tag for all fields, like:

```go
Host string `json:"host" yaml:"host"`
```

Reply:
Ok, get it. BTW, in viper it uses `mapstructure`.

Reply:
fixed

Reply:
Got it.